src/Database/Type/DateTimeType.php

```diff
@@ -52,7 +52,7 @@ class DateTimeType extends Type
      *
      * @var string|array|int
      */
-    protected $_localeFormat;
+    protected $_localeFormat = 'yyyy-MM-dd hh:mm:ss';
```

Comment:
The locale format does not have to be this, it is actually going to be different per locale